spec/v1/providers/database.spec.ts

```diff
@@ -541,17 +541,52 @@ describe('Database Functions', () => {
         expect(subject.val()).to.equal(0);
         populate({ myKey: 0 });
         expect(subject.val()).to.deep.equal({ myKey: 0 });
-
-        // Null values are still reported as null.
-        populate({ myKey: null });
-        expect(subject.val()).to.deep.equal({ myKey: null });
```

Comment:
I've removed this test case as its incorrect compared to the Firebase SDK, the correct version is now in `should deal with null-values appropriately`